test/CMakeLists.txt

```diff
@@ -313,7 +313,7 @@ add_perf_test(test_conv_embed_db
 endif()
 
 if(MIOPEN_TEST_MLIR)
-    set(IMPLICITGEMM_MLIR_ENV_BASE MIOPEN_FIND_MODE=normal)
+    set(IMPLICITGEMM_MLIR_ENV_BASE MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_CONV_WINOGRAD=0)
```

Comment:
That's exactly right. I plan on to also file another ticket to take a look at those "hacks" in NHWC context. Since winograd does not support NHWC, and that the find code force to use winograd in 3x3 cases, it cause unnecessary failures.

Reply:
Hmm... This shouldn't be so. Thanks.

Please also review #905 that fixes #904, if you have time.

Reply:
Please merge recent develop and remove `MIOPEN_DEBUG_CONV_WINOGRAD=0`, thanks.

Reply:
Well, actually you can do that in the next PR. Resolved.

Reply:
Done in e736792